corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -180,7 +180,10 @@ def _filter_invalid_rows_for_form(self, rows, form_id, label_index):
 
     def _get_translation_for_sheet(self, app, sheet_name, rows):
         occurrence = None
-        translations_for_sheet = []
+        # a dict mapping of a context to a Translation object with
+        # clubbed occurrences
```

Comment:
what does "clubbed" mean here?

Reply:
> consolidates translations with same context and source before sending them to transifex 

two translations that have the same source and context would get clubbed together as one with their occurrences merged, for ex:

For example for this module [here](https://india.commcarehq.org/a/myfirsttestproject/apps/view/2374fe2bd7484c06aa80c48f3e168082/module/37fa1ba361352016c277661662eaca964c2e9cf2/#case-detail-screen-config-tab), this should look like this

```
#: name:list:1 name:list:2
msgctxt "name:list"
msgid "Name"
msgstr "Name"

#: name:list:3
msgctxt "name:list"
msgid "Name of Person"
msgstr "Name of Person"
```

the first two occurrences are clubbed together and the two occurrences are mentioned as `name:list:1` and `name:list:2`. The third occurrence is not clubbed because its Display text is different.